examples/settings/set-theme-name.tape

```diff
@@ -0,0 +1,7 @@
+Output examples/settings/theme-name.gif
+Set FontSize 42
+Set Height 225
+Set Theme "Catppuccin Latte"
+
+Type "I can't make up my mind."
+Sleep 1
```

Comment:
This one seems to be ending a bit quickly 🤔

Reply:
fixed, I think I might have press ctrl+c while processing or something